_posts/2017-03-05-artsy-technology-stack-2017.md

```diff
@@ -7,120 +7,101 @@ author: orta
+Most modern code for both the website and the iOS app use an orchestration layer which is powered by [GraphQL](http://graphql.org). Our GraphQL server is an [Express](http://expressjs.com) app, using [express-graphql](https://github.com/graphql/express-graphql) to provide a single API end-point. The API does not access our data directly, but forwards requests to the core API or other services. We have been migrating shared display logic into the GraphQL server, to make it easier to build consistent clients. This [code is open-source](https://github.com/artsy/metaphysics).
 
-https://github.com/artsy/radiation
+Consistently, our front-end code has moved towards using React across all platforms along with introducing stricter JavaScript languages like TypeScript over CoffeeScript in order to provide better tooling.
 
+We continue to have a [public HAL+JSON API](https://developers.artsy.net) for external developers. This API is in active use for a few production services inside Artsy and the [website is open-source](https://github.com/artsy/doppler), too.
 
-[CMS: One for Partners to upload Show, Fair, Artist and Artwork metadata.]
-[Genoming: One for in-house Genomers to handle connecting artworks together.]
-https://github.com/artsy/helix/
+## Partner-Facing
 
-## Auctions
+The vast customer-facing business is powered by a Content Management System (CMS) for gallery and institutional partners. This CMS lets them upload and manager gallery shows, fair booths, create artists, and edit artwork metadata. All CMS components talk to our core API. We also have a number of CMS-like internal applications to manage partners, auctions, art genomes, configuring fairs or performing recurrent billing (we use Stripe for storing and charging credit cards and ACH) with invoicing.
 
-Artsy's Auctions business started with charity auctions. Charity auctions are simpler to map digitally: they have less bids, are more free form in terms of bid increments, have less artworks for sale, they happen slower and the people running them are less risk-averse as they tend to be one-off annual events instead of regular occurrences.
+CMS applications are based on stable, mature technologies like [Rails](http://rubyonrails.org), [Bootstrap](http://getbootstrap.com), [Turbolinks](https://github.com/turbolinks/turbolinks) and [CoffeeScript](http://coffeescript.org), and gradually adopts modern client-side technologies like [React](https://facebook.github.io/react) and [Browserify](http://browserify.org). They share a lot of common infrastructure.
```

Comment:
I might replace Browserify with Webpack here. That's _probably_ a better indication of where we're heading with asset/build management.

Could also add a note about how we're adopting modern JS as a first-class citizen inside Rails apps, both with third-party approaches as well as with an eye to this landing in [core](https://github.com/rails/webpacker) very soon.